src/rates/update.ts

```diff
@@ -63,7 +63,7 @@ export async function updater (): Promise<void> {
           throw Error('Unsupported fiat')
         }
 
-        // @ts-ignore
+        // @ts-ignore: not typed
```

Comment:
Heh lazy you 🤪 This does meet the requirement, but not the reason for that requirement 😉 Some better explanation would be better...

Reply:
Hehe, agree. 
Will fix it.

Reply:
Done